src/io_uring/src/lib.rs

```diff
@@ -63,6 +76,7 @@ pub enum Error {
 }
 
 impl Error {
+    /// Return true if this is error is caused by a full submission queue.
```

Comment:
Did you mean `if this error is caused ...` ?

Reply:
yep, done